tmt/steps/execute/internal.py

```diff
@@ -90,7 +95,7 @@ def options(cls, how: Optional[str] = None) -> List[tmt.options.ClickOptionDecor
             click.option(
                 '--no-progress-bar', is_flag=True,
                 help='Disable interactive progress bar showing the current test.')
```

Comment:
aand I'm missing a comma. This is why you need trailing commas! :smiling_face_with_tear: